### PR TITLE
clean up 

### PR DESCRIPTION
**Pull request by ritumd**
## Checklist before submitting

- [X] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description
spacy 3.2.1 refresh depends on spacy-logger

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: .github/workflows/feedstock-pr.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  optional_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+      - uses: conda-incubator/setup-miniconda@v2.0.0
+        with:
+          auto-update-conda: true
+          python-version: "3.7"
```
Comment (alexlang74) [inline .github/workflows/feedstock-pr.yml:17]: Should we switch to Python 3.9 for these tests? Given that 3.7 is somewhat "old"...

**Review by alexlang74 — APPROVED** [review]

File: .github/workflows/feedstock-pr.yml (hunk 1 above)
Comment (ritumd) [reply .github/workflows/feedstock-pr.yml:17]: Hi @alexlang74  , Post V1.5.1 we shall add this update for python-version 3.9

File: tests/open-ce-test.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+test:
+  - name: Setup spacy-loggers Tests
+    command: |
+        conda install -y pytest>=4.6.5 pytest-timeout mock
+  - name: Run spaCy tests
+        SPACY_LOGGERS_LOCATION=$(python -c "import os; import spacy_loggers; print(os.path.dirname(spacy_loggers.__file__))")
+        pytest -v $SPACY_LOGGERS_LOCATION/tests/
```
Comment (npanpaliya) [inline tests/open-ce-test.yaml:7]: Where is this variable SPACY_LOGGERS_LOCATION defined?

Comment (npanpaliya) [inline tests/open-ce-test.yaml:7]: Where is this variable SPACY_LOGGERS_LOCATION defined?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,26 +12,20 @@ source:
...
-    - wasabi >=0.8.1,<1.1.0
+    - python 
+    - wasabi
 
 test:
   imports:
     - {{ modulename }}
-  requires:
-    - pytest
-    - spacy >=3.1
```
Comment (npanpaliya) [inline recipe/meta.yaml:31]: We removed this test requirement. Should it not be added in run section then?

File: tests/open-ce-test.yaml (hunk 2 above)
Comment (npanpaliya) [reply tests/open-ce-test.yaml:7]: Also need cleanup test section

**Review by npanpaliya — APPROVED** [review]

**Review by cdeepali — APPROVED** [review]

**Pull request merged**